drivers/virtio/Config.uk

```diff
@@ -0,0 +1,2 @@
+config VIRTIO
```

Comment:
Now that `VIRTIO_BUS` is visible, this global option no longer makes sense as it is only useful for `VIRTIO_RING`. So this can be simplified to:

1. Replace this with `ring/Config.uk` that contains a single invisible item named `VIRTIO_RING`
2. Update `ring/Makefile.uk` accordingly
3.  Remove `select VIRTIO` from driver and have them all `select VIRTIO_BUS`.
4. Have `VIRTIO_BUS` to `select VIRTIO_RING`